infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-grafana

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM grafana/grafana:5.4.3
+
+USER root
+ARG TRAFFIC_TS_RPM=traffic_stats/traffic_stats.rpm
+
+RUN apt-get update && apt-get install -y dnsutils net-tools vim tree procps gettext-base curl p7zip-full sudo libcap2-bin
```

Comment:
Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.
You can refer to [docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) and [its source](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105) for details

Reply:
According to the Docker docs:

> _"In addition, when you clean up the apt cache by removing `/var/lib/apt/lists` it reduces the image size, since the apt cache is not stored in a layer."_

You don't need to run `apt-get clean`, but you _should_ run `rm -rf /var/lib/apt/lists/*`

Reply:
got it!